service/nodes.go

```diff
@@ -85,12 +135,31 @@ type exporterNode struct {
 
 func newExporterNode(pipelineID component.ID, exprID component.ID) *exporterNode {
 	return &exporterNode{
-		nodeID:       newNodeID("exporter", string(pipelineID.Type()), exprID.String()),
+		nodeID:       newNodeID(exporterSeed, string(pipelineID.Type()), exprID.String()),
 		componentID:  exprID,
 		pipelineType: pipelineID.Type(),
 	}
 }
 
+func (n *exporterNode) getConsumer() baseConsumer {
+	return n.Component.(baseConsumer)
+}
+
+func (n *exporterNode) build(
+	ctx context.Context,
+	tel component.TelemetrySettings,
+	info component.BuildInfo,
+	builder *exporter.Builder,
+) error {
+	set := exporter.CreateSettings{ID: n.componentID, TelemetrySettings: tel, BuildInfo: info}
+	set.TelemetrySettings.Logger = components.ExporterLogger(set.TelemetrySettings.Logger, n.componentID, n.pipelineType)
```

Comment:
This code is duplicate with the old pipelines code. May worth changing it to:

```
func buildReceiver(ctx context.Context,
	tel component.TelemetrySettings,
	info component.BuildInfo,
	builder *receiver.Builder,
	pipelineID component.ID,
	nexts []baseConsumer,
) (recv component.Component, err error) {
	set := exporter.CreateSettings{ID: n.componentID, TelemetrySettings: tel, BuildInfo: info}
	set.TelemetrySettings.Logger = components.ExporterLogger(set.TelemetrySettings.Logger, n.componentID, n.pipelineType)
	...
}
```

Reply:
[This commit](https://github.com/open-telemetry/opentelemetry-collector/pull/7045/commits/fce05a8fc53b86ed8c98566f776c766b8d5c0feb) contains this change.